src/JCE.Business/Services/Interfaces/ISecurityQuestionsService.cs

```diff
@@ -0,0 +1,8 @@
+﻿using JCE.Business.Dtos.SecurityQuestionsDtos;
+
+namespace JCE.Business.Services.Interfaces;
+public interface ISecurityQuestionsService
+{
+    Task<GetSecurityQuestionsDto> getSecurityQuestions(int userid);
```

Comment:
Try to follow the PascalCase naming convention for methods

Reply:
Corrections done. I adjusted both the profile and the security questions services functions to be written in PascalCase